### DISCUSSION TITLE
Trying to share a vue3 project link

### DISCUSSION THREAD

**Discussion by julianoregisdasilva** [vercel/community#7251] [Help]
### Summary

I have a simple vue3 project created and I would like to deploy it to vercel and it seems this is not working. I'm using my comand line to do so, typing "vercel" and following the instructions, like I show in **image1**. After done, if I go to my projects on vercel site, it appears there, like I show on **image2**. On **image3** I show the project, and I understand it's supposed to have some 'share button' anywhere to allow me to share my project with my team. There is something am I missing? Thanks in advance.
![image1](https://github.com/vercel/community/assets/157991737/b781c7aa-956c-442c-bf14-bbe55e6304e7)
![image2](https://github.com/vercel/community/assets/157991737/6dc71e28-c492-4cd2-b780-07e3dcfa95bf)
![image3](https://github.com/vercel/community/assets/157991737/686bef89-f717-449f-9fff-d9dac9ad7834)

### Steps to Reproduce

.

**Comment by amyegan** [comment]
Hey @julianoregisdasilva! If you visit the logged URLs, you should see the [deployed site](https://boilerplategeorge-50uhap045-juliano-regis-da-silvas-projects.vercel.app/) and [deployment details](https://vercel.com/juliano-regis-da-silvas-projects/boilerplate_george/AqjWiJAA4P4d9d4XDhs6QRFEeXKN).

You should also be able to see the full list of deployments for that project at [vercel.com/juliano-regis-da-silvas-projects/boilerplate_george/deployments](https://vercel.com/juliano-regis-da-silvas-projects/boilerplate_george/deployments). You can [use the `Share` button on a Preview deployment](https://vercel.com/docs/deployments/sharing-deployments) details page to invite other people to view the Preview. 

Production deployments are public by default, so there is no share button for them. You can simply share the assigned domain(s) to let people view the site. 🙂

**Reply by julianoregisdasilva** [reply]
Hey Amy, thanks for your response. You said that "production deployments are public by default...", but when I send the link to anyone, it asks this person to login into vercel, that´s what you talking about? I was expecting to send a link to someone and this person just view my content without need to loging in, is this possible? Second point, you said "You can [use the Share button on a Preview deployment](https://vercel.com/docs/deployments/sharing-deployments) details page to invite other people to view the Preview.". Are you talking about one of the pages which prints I show you now? Can you point me where the 'share' button is? I can't see it. The link you provided drives me to the documentation page. I'm completely lost.
![image1](https://github.com/vercel/community/assets/157991737/b52a1e0c-62ef-4f96-8a03-ad4de68c4b61)
![image2](https://github.com/vercel/community/assets/157991737/49299508-dd2f-4c80-b913-7017cb9aa135)

[a comment by an automated account is not shown]
